taskRequests/details/script.js

```diff
@@ -251,4 +267,162 @@ const renderTaskRequest = async () => {
   }
 };
 
+function showToast(message, type) {
+  toast.innerHTML = `<div class="message">${message}</div>`;
+  toast.classList.remove('hidden');
+
+  if (type === 'success') {
+    toast.classList.add('success');
+    toast.classList.remove('failure');
+  } else if (type === 'failure') {
+    toast.classList.add('failure');
+    toast.classList.remove('success');
+  }
+
+  setTimeout(() => {
+    toast.classList.add('hidden');
+    toast.innerHTML = '';
+  }, 5000);
+}
```

Comment:
Can we  make it like a utility function to be used across the project

Reply:
the `toast` variable used is defined in the `index.html` which is within `task\details` folder. if we move it inside the utility folder then it will not have access to the toast element. correct me if I am wrong.